packages/flutter/lib/src/widgets/reorderable_list.dart

```diff
@@ -1005,6 +1005,7 @@ class ReorderableDragStartListener extends StatelessWidget {
     Key? key,
     required this.child,
     required this.index,
+    this.draggable = true,
```

Comment:
It would be good to add this to the constructor for `ReorderableDelayedDragStartListener` as well in case people run into the same issue but are using the long press drag listener. Which means we should also have a test for that constructor as well.

Reply:
Sure, that is a valid point ! I'll do it in my next commit